core/src/main/java/org/dozer/config/processors/DefaultSettingsProcessor.java

```diff
@@ -43,7 +43,6 @@ public Settings process() {
         createSettingsResolvers();
         checkForDeprecatedKeys();
 
-        Boolean elEnabled = Boolean.valueOf(getValue(SettingsKeys.EL_ENABLED, SettingsDefaults.EL_ENABLED).toString());
```

Comment:
Maybe we should keep the possibility to set the ELEngine via config file? Just like with proxy resolver and class loader, this could be another bean.

Reply:
The reason i didn't want to do this, was because of class loader issues.